### PR TITLE
Alpha feb2023

### PR DESCRIPTION
**Pull request by mshukla1**

### CODE REVIEW

**Comment by olsonanl** [comment]
I don't think the second removals of display:none are correct. the code at https://github.com/BV-BRC/BV-BRC-Web/blob/master/public/js/p3/widget/app/MetagenomicReadMapping.js#L373 manages the visibility of those widgets based on the value of the radio buttons, and the default states appear to be correct. I've not tried the code yet on my own however to verify this.

File: public/js/p3/widget/app/templates/MetagenomicReadMapping.html [hunk 1]
```diff
@@ -95,7 +95,7 @@ <h3>${applicationLabel}
             </div>
           </div>
         </div>
-        <div class="appRow" style="display:none">
+        <div class="appRow">
```
Comment (chris-c-thomas) [inline public/js/p3/widget/app/templates/MetagenomicReadMapping.html:98]: Keep this to display the radio buttons

File: public/js/p3/widget/app/templates/MetagenomicReadMapping.html [hunk 2]
```diff
@@ -113,13 +113,13 @@ <h3>${applicationLabel}
             </select>
           </div>
         </div>
-        <div data-dojo-attach-point = "geneSetFastaWidgetBox" style="display:none">
+        <div data-dojo-attach-point = "geneSetFastaWidgetBox">
```
Comment (chris-c-thomas) [inline public/js/p3/widget/app/templates/MetagenomicReadMapping.html:116]: Set back to display: none as handled by the logic

File: public/js/p3/widget/app/templates/MetagenomicReadMapping.html [hunk 3]
```diff
@@ -113,13 +113,13 @@ <h3>${applicationLabel}
...
           <div class="appRow">
             <label class="paramlabel">Gene Set Fasta</label><br>
             <div data-dojo-type="p3/widget/WorkspaceObjectSelector" data-dojo-attach-point="gene_set_fasta" name="gene_set_fasta" required="false"  data-dojo-props="type:['feature_protein_fasta'],multi:false,promptMessage:'Select or Upload protein fasta file to your workspace', placeHolder:''"></div>
           </div>
         </div>
-        <div data-dojo-attach-point = "geneSetFeaturegroupWidgetBox" style="display:none">
+        <div data-dojo-attach-point = "geneSetFeaturegroupWidgetBox">
```
Comment (chris-c-thomas) [inline public/js/p3/widget/app/templates/MetagenomicReadMapping.html:122]: Set back to display: none as handled by the logic

**Review by chris-c-thomas — CHANGES_REQUESTED** [review]
@olsonanl is right. Just have the radio buttons display on line 98 and the logic works as expected.

**Review by olsonanl — APPROVED** [review]

**Review by olsonanl — APPROVED** [review]

**Pull request merged**